src/redux/actions/group-actions.js

```diff
@@ -178,6 +178,7 @@ export const removeMembersFromGroup = (groupId, members) => {
 export const fetchRolesForGroup = (groupId, pagination, options) => ({
   type: ActionTypes.FETCH_ROLES_FOR_GROUP,
   payload: GroupHelper.fetchRolesForGroup(groupId, false, pagination, options).catch(handleUuidError),
+  meta: { groupId },
```

Comment:
you could pass something like `storeInGroups` boolean flag to switch between adding the data to `roles` or `selectedRole`. The same applies for members. It should also help you manage the loading boolean flag. Creating another actions for fetching expandable tables data is probably also correct if you want to go that way